function-sidecar/cmd/function-sidecar.go

```diff
@@ -163,8 +168,12 @@ func backoff() bool {
 	} else {
 		// Back off a bit to give the invoker time to come back
 		// (if we support windowing or polling this logic will be more complex)
-		time.Sleep(1000 * time.Millisecond)
-		return true
+		if backoffRetries > 0 {
+			backoffDuration = backoffDuration * time.Duration(backoffMultiplier)
+		}
+		time.Sleep(backoffDuration * time.Millisecond)
+		backoffRetries++
+		return backoffRetries < maxBackoffRetries
```

Comment:
Is there a way to write something in go like `return backoffRetries++ < maxBackoffRetries` ?

Reply:
No. In Go, the `++` and `--` operators form statements, not expressions.